systemd/os-autoinst-scripts-update-git.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update git repository of os-autoinst-scripts
+
+[Timer]
+OnCalendar=*:0/3
+Unit=os-autoinst-scripts-update-git.service
+
+[Install]
+WantedBy=default.target
```

Comment:
I think the default target should be
```suggestion
[Install]
WantedBy=timers.target
```

Reply:
I've seen both 🤔 Not sure what the difference is in practice. I don't see an explicit mention of that in [systemd timer upstream docs](https://man.archlinux.org/man/systemd.timer.5#OPTIONS) or systemd.unit.

Reply:
https://www.freedesktop.org/software/systemd/man/latest/systemd.special.html#timers.target